lib/cisco_node_utils/cmd_ref/vlan.yaml

```diff
@@ -11,6 +11,20 @@ create:
 destroy:
   config_set: "no vlan %s"
 
+feature_vn_segment:
```

Comment:
It appears that we already have a yaml entry for this in `feature.yaml`. 
- See https://github.com/cisco/cisco-network-node-utils/blob/release_1.2.0/lib/cisco_node_utils/cmd_ref/feature.yaml#L36.

These yaml entries are called by our new node_utils class that manages various features. (We intend to move all nxos features into this) 
- https://github.com/cisco/cisco-network-node-utils/blob/8b52c2abb51cad5dacc59593d8fe214cd146a410/lib/cisco_node_utils/feature.rb

Here is an example of how it's called in the `vxlan_vtep` NU object.
- https://github.com/cisco/cisco-network-node-utils/blob/dbddc357b343c524c12c5c7944a735854a6e020b/lib/cisco_node_utils/vxlan_vtep.rb#L59